SentinelOne/cloud_funnel2.0/ingest/parser.yml

```diff
@@ -275,7 +275,7 @@ stages:
         filter: '{{ convert_process_start != null and json_event.message.get("osSrc.process.pid") == None}}'
 
       - set:
-          deepvisibility.process.activecontent.path: '{{json_event.message.get("src.process.activeContent.path")}}'
```

Comment:
Can we keep `deepvisibility.process.activecontent.path` in addition to `action.properties.Path`? 
 the field is used.